src/main/java/uk/gov/justice/framework/command/tools/SystemCommanderClientFactoryHelper.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.framework.command.tools;
+
+import uk.gov.justice.services.jmx.system.command.client.SystemCommanderClientFactory;
+
+public class SystemCommanderClientFactoryHelper {
```

Comment:
do we need new SystemCommanderClientFactory() every time, or we can use singleton pattern?